### PR TITLE
rm readonly + notice + db2 from files

### PR DESCRIPTION
**Pull request by MacFan4000**

### CODE REVIEW

File: Database.php [hunk 1]
```diff
@@ -3,7 +3,6 @@
 $wgLBFactoryConf = array(
 	'class' => 'LBFactoryMulti',
 	'sectionsByDB' => array(
-	        // 'wiki' => 'c1'
```
Comment (paladox) [inline Database.php]: Re add this please.

File: LocalSettings.php [hunk 2]
```diff
@@ -6179,7 +6179,7 @@ function onSiteNoticeAfter( &$siteNotice, $skin ) {
 	return true;
 }
 */
-
+/*
 // leaving here as placeholder for future db work
 if ( $wgDBname == 'airwiki' ||
```
Comment (paladox) [inline LocalSettings.php]: Remove all these but keep two for examples please.

Comment (JohnFLewis) [reply LocalSettings.php]: or remove them all because we already have a site notice thing commented out

File: Database.php (hunk 1 above)
Comment (MacFan4000) [reply Database.php]: This isn’t needed

Comment (paladox) [reply Database.php]: Yes it is for reference as we need docs.

**Pull request merged**